site/contents/write_ups/2022/12_01_22-writeup.md

```diff
@@ -0,0 +1,93 @@
+
+The summary was written by Huw Day, who tried to synthesise everyone's contributions to this document and the discussion. "We" = "someone at Data Ethics Club". 
+Nina Di Cara and Natalie Thurlby helped with the final edit.
+
+```
+
+## Happy Birthday to Data Ethics Club!
+
+This meeting marks a whole year of Data Ethics Club and so we decided to do something a little bit different, especially as it lined up with New Years. We have spent the past year discussing and debating various issues within data ethics and adjacent fields. With many of us coming from academic backgrounds, we can find it easy to spend a lot of time complaining without actually addressing any issues. In this talk, we talked about the ways we (both as individuals and as a collective) can make our little dents in addressing some of these issues we have discussed, debated and learnt about over the past year.
+
+## Why is what we do important?
+
+The rules of arithmetic are objective, but the way we use them are not, which leads to inevitable divide between theory in practise. We have to make important decisions about where we spend our money, where we have our conferences and where we put our attention. Data science is just a tool to help us make these decisions.
+
+Maybe the question is 'what are we going to commit to **not** doing' or 'what are we going to call other people out for doing?'. How do we stand up to your superiors, and justify refusing to do something? This is especially difficult in commerical environments. We have found that, as individuals, it is difficult to stick your head above the parapet - you need to have a bit of a community to fall back on. Building a community where you can discuss ethical issues and fall back to is vital for getting more people to act more ethically, even if it is an uphill battle sometimes.
+
+How can we be activists? Making statements to the university etc. Do we want to consider a mission statement? Do we want to encourage development of ethical courses?
+We as a group should think about what kind of impact do we want to have? Recommendations, running courses, 
+
+Data ethics club has the potential to act as a support group. We do not necessarily have all the same views but having data ethics as an important component of our professional practice brings us together. Communities are important to help us improve our own behaviour, holding us accountable whilst supporting us through changes.
+
+A lot of the problems we have discussed over the past year have arrived because there has been nobody present to question people's ethics. Ethics always pertains to structures of power, vested interests and conflicts that pertain to these things. The whole point of data ethics club is to advocate for data science practitioners to practise their craft in a more ethically sound way. Of course, we don't want just a focus on STEM. We need people to be ethical in using data across other fields such as sociology and philosophy.
```

Comment:
```suggestion
A lot of the problems we have discussed over the past year have arrived because there has been nobody present to question people's ethics. 
Ethics always pertains to structures of power, vested interests and conflicts that relate to these things. 
The whole point of Data Ethics Club is to advocate for data science practitioners to practise their craft in a more ethically sound way.
Of course, we don't want just a focus on STEM. 
We need people to be ethical in using data across other fields such as sociology and philosophy, and to allow those fields to influence data science.
```